frontends/election-manager/src/screens/settings_screen.tsx

```diff
@@ -56,13 +56,13 @@ export function SettingsScreen(): JSX.Element {
           />{' '}
           or <RebootToBiosButton logger={logger} />
         </p>
-        <h2>USB Sticks</h2>
+        <h2>USB Drives</h2>
         <p>
           <Button
             disabled={!formatButtonEnabled}
             onPress={() => setShowFormatModal(true)}
           >
-            Format USB Stick
+            Format USB drive
```

Comment:
I believe we use Title Case for button text, so "Format USB Drive" here (unless we've since changed convention)